tests/workbenches/docs.py

```diff
@@ -0,0 +1,39 @@
+from typing import Callable
```

Comment:
What is this used for? does it just provide additional documentation for tests in e.g. https://github.com/opendatahub-io/opendatahub-tests/pull/94/files#diff-358f1122838e1ac22415822c5a85d419e2d111c64285e488695af1cf6f03d490R35 ? 
IMHO if this is implemented we should use it across the repo to keep consistency, so this should go under a higher level directory, but it should probably be discussed beforehand. Can you attend the shift left wg meeting to do so?

Reply:
I'm not sure how useful this docs is going to be. Yes, it is only for structured comments. The Java version was able to render markdown docs from this, but I did not implement this in Python here.

IMO it's fine for now to only have this in the workbenches tests, and we (in workbenches team) need to decide if this is what we want to promote, or if we decide to delete this eventually.

Reply:
ok then